modules/typesniffer/typesniffer.go

```diff
@@ -97,6 +98,16 @@ func DetectContentType(data []byte) SniffedType {
 		ct = SvgMimeType
 	}
 
+	if strings.HasPrefix(ct, "audio/") && bytes.HasPrefix(data, []byte("ID3")) {
+		// the MP3 detection is quite inaccurate, any content with "ID3" prefix will result in "audio/mpeg"
+		// so remove the "ID3" prefix and detect again, if result is text, then it must be text content.
+		// This works especially because audio files contain many unprintable/invalid characters like "\000" 
```

Comment:
Looks like `\000` in comment triggered some bug of `gitea-fmt`.

Reply:
Hack it, bypass! I am really good at hacky pathces .... 😂